invenio_cli/cli/packages.py

```diff
@@ -67,11 +68,28 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages(packages)
+
+    on_fail = f"Failed to show outdated packages."
+    on_success = f"Outdated packages."
+
+    run_steps(steps, on_fail, on_success)
+
+    os.popen('pipenv update --outdated')
```

Comment:
major: This is something left out right? The command is being executre in the `run_steps`... if not, it should 😅